test/hotspot/gtest/classfile/test_placeholders.cpp

```diff
@@ -39,10 +39,10 @@ TEST_VM(PlaceholderTable, supername) {
   ThreadInVMfromNative tivfn(THREAD);
 
   // Assert messages assume these symbols are unique, and the refcounts start at one.
-  TempNewSymbol A = SymbolTable::new_symbol("abc2_8_2023_class");
-  TempNewSymbol D = SymbolTable::new_symbol("def2_8_2023_class");
+  Symbol* A = SymbolTable::new_symbol("abc2_8_2023_class");
+  Symbol* D = SymbolTable::new_symbol("def2_8_2023_class");
   Symbol* super = SymbolTable::new_symbol("super2_8_2023_supername");
-  TempNewSymbol interf = SymbolTable::new_symbol("interface2_8_2023_supername");
+  Symbol* interf = SymbolTable::new_symbol("interface2_8_2023_supername");
 
```

Comment:
I swapped these from TempNewSymbol to Symbol before I added the set_cleanup_delay_max_entries functionality to avoid interfering with refcounts in tests.
Is the swap safe? Or should I use set_cleanup_delay_max_entries(0) and switch back? I can't tell why some of these were made temp and some not.

Reply:
I don't remember why they were TempNewSymbol either but I don't think it matters for this test.

Reply:
I still don't think it matters for the test, but this seems to be an effective workaround.

Reply:
That's a better idea, thanks!